include/librealsense2/h/rs_option.h

```diff
@@ -76,6 +76,10 @@ extern "C" {
         RS2_OPTION_HARDWARE_PRESET, /**< Hardware stream configuration */
         RS2_OPTION_GLOBAL_TIME_ENABLED, /**< disable global time  */
         RS2_OPTION_APD_TEMPERATURE, /**< APD temperature*/
+        RS2_OPTION_ENABLE_MAPPING, /**< Enable an internal map */
```

Comment:
The new features/usage also should be elaborated in [Readme](https://github.com/IntelRealSense/librealsense/blob/development/doc/t265.md)